x-pack/plugins/cases/server/connectors/cases/utils.ts

```diff
@@ -78,3 +78,7 @@ export const buildRequiredCustomFieldsForRequest = (
         })
     : [];
 };
+
+export const constructRequiredKibanaPrivileges = (owner: string): string[] => {
+  return [`cases:${owner}/createCase`, `cases:${owner}/updateCase`];
```

Comment:
I guess for case action user doesn't need to have delete permission, right?

Reply:
Good point. At the moment the case action does not do any delete operation but I will add it just to be safe in case we need it in the future.